codelists/forms.py

```diff
@@ -105,6 +107,12 @@ def clean_csv_data(self):
                     for i in errors
                 ]
             )
+        cleaned_header = [
+            "".join(re.match(r"(\"?)\s*(\w+)\s*(\"?)", h).groups()) for h in header
```

Comment:
nb. again I have to lean on your knowledge of the source data - obviously that `(\w+)` won't match any space chars - so if your header was two words `" my data "` it would not match. Hopefully they're all one word & this is fine?

Reply:
That's another good catch! I think we need to allow for it, but I wasn't thinking about that when I wrote that pattern. I'll fix it.